src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -167,6 +168,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	let downloadedPackagesStore: DownloadedPackagesStore = DownloadedPackagesSQLLiteStore(fileName: "packages")
 
 	let client: HTTPClient
+	let wifiClient: HTTPClientWifiOnly
```

Comment:
To stay consistent I would propose the name: WifiOnlyHTTPClient. 
Its like CachingHTTPClient.

Reply:
good point - I changed it